### ISSUE TITLE
Is the Tekton Helm chart published?

### ISSUE THREAD

**Issue by tlawrie** [tektoncd/operator#873]
Is the Tekton helm chart published in a chart repository? currently, the instructions point to cloning the git repository and installing the helm chart from there which isn't entirely useful for automated installs, especially if a dependent chart to another helm chart.

**Comment by sm43** [comment]
@jacksgt any update when we would be able to publish the chart?
cc @vdemeester

**Comment by tlawrie** [comment]
Thanks @sm43. Any update on the ETA for this?

**Comment by sm43** [comment]
@tlawrie sorry for the delay.
@jacksgt has contributed the helm chart to the project, just waiting for him to give a go.
then we can work on publishing the chart :)

**Comment by tlawrie** [comment]
Hi @sm43 @jacksgt how are we looking?

**Comment by sm43** [comment]
@tlawrie I am looking into helm, it is a bit new for me 😅 
will try to get this done soon :)

**Comment by tlawrie** [comment]
Do you have any questions? I've done a helm chart in the past.

Is it in figuring out where to package and host it?

**Comment by sm43** [comment]
> Do you have any questions? I've done a helm chart in the past.
> 
> Is it in figuring out where to package and host it?

Yeah Currently, I am updating the chart in main branch for the latest released version of operator 
and once it works fine was thinking to setup github action for the release
https://github.com/marketplace/actions/helm-chart-releaser
WDYT?

**Comment by jacksgt** [comment]
Hey folks, thanks for the interest and sorry for the long delay!

I created a PR a while ago that should automatically update the Helm chart for each new version of the operator: https://github.com/tektoncd/operator/pull/816
Unfortunately, so far no one has tested that PR :-/

Once we have that in place, using the helm-chart-releaser action as suggested above should be fairly straightforward.

**Comment by sm43** [comment]
> I created a PR a while ago that should automatically update the Helm chart for each new version of the operator: #816 Unfortunately, so far no one has tested that PR :-/

hey @jacksgt sorry for that. The PR has a work in progress label due to which I didn't take a look earlier. Let me take a look.

**Comment by tlawrie** [comment]
Checking back in on this? any movement?

**Comment by tlawrie** [comment]
Following up on this. Any update?

**Comment by sm43** [comment]
hey @tlawrie #816 might take time to get merged. automation will take some time to setup as I don't access to the release cluster. 

so till then we will update the chart manually and https://github.com/tektoncd/operator/pull/1006 is for creating a release.
we will be releasing 0.62.0 today or on monday. once that is done, I will update the chart for it.

please feel free to contribute any improvements to chart or chart release workflow. 🙃

**Comment by sm43** [comment]
https://github.com/tektoncd/operator/releases/tag/tekton-operator-chart-0.61.0

**Comment by brovoca** [comment]
Any update on this? Additionally, 0.62.0 from September is not in the chart.

**Comment by ChrisJBurns** [comment]
I'm more than happy to help with the Helm Chart. I've done a few over the last couple of months for a couple of OSS projects that had less than ideal installation guides. I'd normally offer to do the standard manifests that users use to install Tekton but as the operator is the recommended way of installing and managing Tekton I'm more than happy to contribute to this as I really like Tekton and I do think that the only thing holding it back from widespread adoption is the complexity of it. 

If I can help reduce that complexity of installation into usable Helm Charts, I'd be more than happy to do this. Additionally, I'd like to also implement TektonChains into the chart also? As for me this is the powerful aspect of Tekton that separates it from other CI tools. Let me knows your thoughts.

**Comment by jacksgt** [comment]
Hey folks,

this is currently blocked on https://github.com/tektoncd/operator/issues/1330

We already have a Helm chart: https://github.com/tektoncd/operator/tree/main/chart

and also automated CI for publishing the chart: https://github.com/tektoncd/operator/blob/main/.github/workflows/helm-release.yaml

As mentioned in the issue above, there is currently a minor permission issue with the CI and the repository needs to enable GitHub Pages - then we're good to go!

**Comment by brovoca** [comment]
/reopen

A working solution has not been announced here yet.

**Comment by bdellegrazie** [comment]
Can someone with permissions please re-activate issue #1330 *and mark it fresh*... it seems a shame to bury all that work when the only thing blocking publication of the helm charts is permissions and an overzealous bot.

**Comment by bdellegrazie** [comment]
Raised #1915 which augments this effort by allowing the repository to be used directly via the [helm-git](https://github.com/aslafy-z/helm-git) plugin.

**Comment by aljoshare** [comment]
Is there any news? 😇

**Comment by jefferson-lima** [comment]
Hi folks, thanks for the effort of trying to implement this, I just would like to say that this feature would be much appreciated.

**Comment by sillock1** [comment]
Any chance we can actually get this chart published to either github pages or ideally into an OCI registry like ghcr?
edit: Although the chart isn't published to the github.io static site, you can still use the helm chart without the helm-git plugin, with GitOps tools like flux.
Use the raw link for the helm repository https://raw.githubusercontent.com/tektoncd/operator/gh-pages, it works fine

**Comment by kastl-ars** [comment]
> edit: Although the chart isn't published to the github.io static site, you can still use the helm chart without the helm-git plugin, with GitOps tools like flux. Use the raw link for the helm repository https://raw.githubusercontent.com/tektoncd/operator/gh-pages, it works fine

I was looking for a way to install the chart that does not involve manual hacks. And found your comment, thankfully.

Honestly I am a little puzzled on why creating and publishing a chart is still not finished after almost 4 years. I understand that all projects are missing contributors and maintainers, but setting up chart publishing is not that big of a task. Especially as the chart seems to be in place and updated, it is just the "official" publishing that is missing.
